src/schemas/json/function.json

```diff
@@ -514,6 +516,59 @@
           "description": "The text of the email."
         }
       }
+    },
+    "sqlBinding": {
+      "properties": {
+        "connectionStringSetting": {
+          "type": "string",
+          "description": "The name of the app setting that contains the SQL connection string used to connect to the database."
+        }
+      },
+      "oneOf": [
+        {
+          "properties": {
+            "type": {
+              "const": "sqlTrigger"
+            },
+            "tableName": {
+              "type": "string",
+              "description": "The name of the table to be monitored for changes."
+            }
+          }
+        },
+        {
+          "properties": {
+            "type": {
+              "const": "sql"
+            },
+            "direction": {
+              "const": "in"
+            },
+            "commandText": {
+              "type": "string",
+              "description": "Either a query string or the name of a stored procedure to execute."
+            },
+            "commandType": {
+              "enum": ["text", "storedProcedure"],
+              "description": "Whether the commandText is a query (text) or a stored procedure (storedProcedure)."
+            }
+          }
+        },
+        {
+          "properties": {
+            "type": {
+              "const": "sql"
+            },
+            "direction": {
+              "const": "out"
+            },
+            "commandText": {
```

Comment:
Can we name it `tableName`?

Reply:
That's something I want to follow up with later on for possibly changing, but for now it's commandText (that's the value used in the attribute). If we change it in the attribute, I'll update the schema here too.